src/solvers/dg_p4est/containers.jl

```diff
@@ -100,13 +100,51 @@ end
 @inline nboundaries(boundaries::BoundaryContainerP4est) = length(boundaries.element_ids)
 
 
+struct MortarContainerP4est{NDIMS, uEltype<:Real, NDIMSP1, NDIMSP3} <: AbstractContainer
+  u::Array{uEltype, NDIMSP3}       # [small/large side, variable, position, i, j, mortar]
+  element_ids::Matrix{Int}         # [position, mortar]
+  node_indices::Matrix{NTuple{NDIMS, Symbol}} # [small/large, mortar]
```

Comment:
How does this mortar container work? Is it one mortar per element or one mortar per "non-conforming surface", like in the TreeMesh?

Reply:
Yes. I planned on adding documentation here, but I forgot to do it in this PR.

Reply:
OK, can you please add it here?

Reply:
Or one that takes only two arguments (is that possible for a struct with 4 arguments?) and computes `NDIMS + 1` and `NDIMS + 3` itself?
But I don't think that's necessary here. This is the only function that will ever construct a new container.

Reply:
Okay, ignore this for now :+1: